mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/90 - scenario details view

Closes #90 

Also did a big refactor of all the factory SDF file parsing to reduce duplicate code by having them all use new functions in ccUtils.

To test:
- Play any scenario
- Click button in upper left corner
- Make sure scenario briefing displays correctly

Notes:
- Restart buttons aren't implemented because the server doesn't have that functionality yet (I'll make a separate case for those)
- Save button just makes a .sdf file in the user AppData folder, not sure if this servers any purpose
